src/clr-angular/data/datagrid/datagrid-row.ts

```diff
@@ -78,6 +79,7 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
     nbRow++;
     this.id = 'clr-dg-row' + nbRow;
     this.radioId = 'clr-dg-row-rd' + nbRow;
+    this.checkboxId = 'clr-dg-row-cb' + nbRow;
```

Comment:
Does this need a spec?

Reply:
There weren't any for the radio before, so added both.